src/fmu/sumo/explorer/_utils.py

```diff
@@ -14,21 +29,197 @@ class Property(str, Enum):
+    logger.debug("hits: %s", len(hits))
+    logger.debug(hits)
+    return_count = len(hits)
+    if return_count < total_count:
+        message = (
+            "Your query returned less than the total number of hits\n" +
+            f"({return_count} vs {total_count}). You might wanna rerun \n" +
+            f"the query with size set to {total_count}"
+        )
+        warnings.warn(message, TooLowSizeWarning)
+    return hits
+
+
+def get_vector_name(source):
+    """Gets name of vector from query results
+    source (dict): results from elastic search query _source
+    """
+    name = source["data"]["spec"]["columns"][-1]
+    return name
+
+
+def deal_w_tag(kwargs):
+    """Works on what to do with tag
+    kwargs (dict): dictionary"""
+    print(kwargs)
+    name_of_tags = "tagname"
+    tagname = kwargs.get(name_of_tags, None)
+    standard_get_name = True
+    if tagname is not None:
+        del kwargs[name_of_tags]
+    if (kwargs["data_type"] == "table" and
+        kwargs["content"] == "timeseries"):
+        standard_get_name = False
+    return tagname, standard_get_name
+
+
+def perform_query(case, **kwargs):
+    """Performs an elastic search
+    args:
+    case (explorer.Case): case to explore
+    kwargs (dict): keword argument
+    """
+    logger = init_logging(__name__ + ".get_object_blobs")
+    logger.debug("Calling function with %s", kwargs)
+    convert = {"data_type": "class", "content": "data.content",
+               "name": "data.name", "tag": "data.tagname",
+               "iteration": "fmu.iteration.id"}
+
+    size = kwargs.get("size", 10)
+    try:
+        del kwargs["size"]
+    except KeyError:
+        logger.debug("Size is predefined")
+
+    query = f"_sumo.parent_object:{case.sumo_id}"
+    for key, value in kwargs.items():
+        if key == "tag":
+            # Somehow the tagname doesn't work
+            continue
+        query += f" AND {convert[key]}:{value}"
+    logger.debug("sending query: %s", query)
+    results = return_hits(case.sumo.get("/search", query=query, size=size))
+    return results
+
+
+def get_object_blobs(case, **kwargs):
+    """Makes dictionary pointing to blob files
+    args:
+    case (explorer.Case): case to explore
+    kwargs (dict): keword argument
+    return blob_dict (dict): dictionary of blobs, key is name
+                             value is blob path
+    """
+    logger = init_logging(__name__ + ".get_object_blobs")
+    print(kwargs)
+    tagname, standard_get_name = deal_w_tag(kwargs)
+
+    results = perform_query(case, **kwargs)
+    logger.debug(len(results))
+    blobs = {}
+    for result in results:
+        source = result["_source"]
+        if tagname is not None:
+            if source["data"]["tagname"] != tagname:
+                continue
+        if standard_get_name:
+            name = source["fmu"]["realization"]["id"]
+        else:
+            name = get_vector_name(source)
+
+        blobs[name] = source["_sumo"]["blob_url"]
```

Comment:
This may not be useful, as the `blob_url` does not contain an authorization token.

Reply:
Fixed: Now the function is renamed to get_object_blob_ids, and corresponding methods on class Case are renamed to get_blob_ids, and get_summary_blob_ids. All are tested with unit tests, and towards a webviz plugin.

Reply:
And line changed to: blob_ids[name] = result["_id"]